src/animation/friction.ts

```diff
@@ -92,7 +89,10 @@ export const friction1D = (
         position: state.mover.position,
       });
     }
-  });
+  };
+
+  const { start } = rAF();
+  const runAnimation = () => start(listener);
 
-  return { controller: { start, stop } };
+  return { controller: { start: runAnimation } };
```

Comment:
Rather than calling `start` inside of this function, we now return a function that calls `start` with the `listener` already bound. This is what allows us to attach `start` to the `controller` for our users to use.